src/bot.ts

```diff
@@ -106,6 +107,24 @@ Also, here are some links you might find useful:
   )
 }
 
+function _getTokenFmt (amount: BigNumber, token: TokenDto) {
+  let tokenLabel, tokenParam
+  if (token.known) {
+    tokenLabel = token.symbol || token.name || token.address
+    tokenParam = tokenLabel
```

Comment:
This might not work for tokens that have `name` and not have `symbol`. You should consider only `symbol` and `address` for `tokenParam`.

Reply:
@anxolin guess you missed this comment

Reply:
In reality, if it's in the list it will have symbol for sure, but I agree with you that it should be only symbol or address here, especially if we use a TCR controlled by 3rd parties

I'll change it to, let me know if it's better:
```js
tokenLabel = token.symbol || token.name || token.address
tokenParam = token.symbol || token.address
```